examples/expressions/13_timeseries_forecasting.py

```diff
@@ -0,0 +1,488 @@
+
+.. |TimeSeriesSplit| replace::
+    :class:`~sklearn.model_selection.TimeSeriesSplit`
+
+.. |ColumnTransformer| replace::
+    :class:`~sklearn.compose.ColumnTransformer`
+
+.. |make_column_transformer| replace::
+    :class:`~sklearn.compose.make_column_transformer`
+
+.. |RidgeCV| replace::
+    :class:`~sklearn.linear_model.RidgeCV`
+
+.. |SimpleImputer| replace::
+    :class:`~sklearn.impute.SimpleImputer`
+
+.. |StandardScaler| replace::
+    :class:`~sklearn.preprocessing.StandardScaler`
+
+.. |ToDatetime| replace::
+    :class:`~skrub.ToDatetime`
+
+.. |var| replace::
+    :meth:`skrub.var`
+
+.. |apply_func| replace::
+    :meth:`skrub.Expr.skb.apply_func`
+
+.. |choose_bool| replace::
+    :meth:`skrub.Expr.skb.choose_bool`
+
+.. |choose_from| replace::
+    :meth:`skrub.Expr.skb.choose_from`
+
+.. |train_test_split| replace::
+    :meth:`skrub.Expr.skb.train_test_split`
+
+.. _scikit-learn example:
+ `<https://scikit-learn.org/stable/auto_examples/applications/plot_time_series_lagged_features.html#generating-polars-engineered-lagged-featurehttps://scikit-learn.org/stable/auto_examples/applications/plot_time_series_lagged_features.html#generating-polars-engineered-lagged-featuress>`_
+
+"""
+
+# %%
+# A problem with relevant datetime features
+# -----------------------------------------
+#
+# We will use a dataset of bike sharing demand in 2011 and 2012.
+# In this setting, we want to predict the number of bike rentals, based
+# on the date, time and weather conditions.
+# In this example, we will use Polars dataframes instead of Pandas.
+
+
+import polars as pl
+
+from skrub import TableReport, datasets
+
+data = datasets.fetch_bike_sharing().bike_sharing
+data = pl.from_pandas(data)
+TableReport(data)
+
+
+# %%
+# Prediction with datetime features
+# ---------------------------------
+#
+# In this section, we will use skrub Expressions to build a predictive pipeline
+# that pre-processes the data and performs hyperparameter optimization for various
+# of the steps involved in the preparation.
+# We will use a |RidgeCV| model as our learner. While it is not the most powerful
+# model, it will help us illustrate how feature engineering affects the prediction
+# performance.
+
+
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import RidgeCV
+
+###############################################################################
+# Since we are working with timeseries, we need to use |TimeSeriesSplit| to perform
+# crossvalidation.
+from sklearn.model_selection import TimeSeriesSplit
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+ts = TimeSeriesSplit(
+    n_splits=3,  # to keep the notebook fast enough on common laptops
+    gap=48,  # 2 days (48 hours) data gap between train and test
+    max_train_size=10000,  # keep train sets of comparable sizes
+    test_size=3000,  # for 2 or 3 digits of precision in scores
+)
+
+import skrub
```

Comment:
Those imports look weird at the end of the cell. Maybe on the top of where they are going to be used.